test/verify/files/ssh/id_ed25519

```diff
@@ -1,8 +1,8 @@
 -----BEGIN OPENSSH PRIVATE KEY-----
-b3BlbnNzaC1rZXktdjEAAAAACmFlczI1Ni1jYmMAAAAGYmNyeXB0AAAAGAAAABCOitCaLN
-rylREgwxd1e8NuAAAAEAAAAAEAAAAzAAAAC3NzaC1lZDI1NTE5AAAAIEFGhZ5twoV3DEA4
-yRIsTvF7t8xFKcj8qHW4PJKbKyFTAAAAkAv9ozmldrTPS+ru1mOI8JnfDA4P7e/jdviaXQ
-TbTBigljz/ZoOReJtBO/2PHbtgm2ekM8UdTnB3VWjxLSH+z7UpgVHtqnuN4PQ+Rueq1kMq
-AoAamod8u1gb7RL9tcxGBRRrDMoGnO/f3QZIOEKDvqKSDy/PXB+Lmh4od1nmCQBhMsTMWk
-1hSB9/iF9UGn+mvw==
+b3BlbnNzaC1rZXktdjEAAAAACmFlczI1Ni1jdHIAAAAGYmNyeXB0AAAAGAAAABBxFawn02
```

Comment:
How did that file get changed? Changing the password from "foobar" to "locked"? Why was that done, do we use the login password to try and unlock ssh keys? (that sounds a bit weird). The commit message should point that out.

Reply:
Yea if the password is the same as the user password, it'll just automatically get added to the keyring. I'm not sure at what point that happens (for sure before cockpit-ssh gets invoked of course).